### ISSUE TITLE
.tar.bz2 artifacts seem to be incomplete

### ISSUE THREAD

**Issue by msarahan** [fribidi/fribidi#76]
I'm attempting to build fribidi with meson, based on the 1.0.2 tar.bz2 on your releases page, and hitting this error:

```
test/meson.build:16:2: ERROR: File test_CapRTL_isolate.input does not exist.
```

Looking in the source code extracted from the .tar.bz2 I got from your releases page, sure enough, that file does not exist (though I do see it in the github repo)

```
Makefile.am                     test_CapRTL_explicit.reference  test_UTF-8_persian.reference
Makefile.in                     test_CapRTL_implicit.input      test_UTF-8_reordernsm.input
meson.build                     test_CapRTL_implicit.reference  test_UTF-8_reordernsm.reference
run.tests                       test_ISO8859-8_hebrew.input     unicode-conformance
test-runner.py                  test_ISO8859-8_hebrew.reference
test_CapRTL_explicit.input      test_UTF-8_persian.input
```
How are your .tar.bz2 files created?  Are there scripts that I could submit a PR to?

**Comment by dov** [comment]
Indeed, those files were added to the repo in commit:5607baae , which is later than the  v1.0.2 release. I guess it's time for v1.0.3 ...

**Comment by msarahan** [comment]
Strangely, checking out the git repo, then checking out the v1.0.2 tag leaves me with a working collection of files.  Probably an artifact of how git handles new files.  Anyway, thanks for taking a look and bumping to capture the current set.

**Comment by lazka** [comment]
In case someone needs a patch like I did :) https://github.com/Alexpux/MINGW-packages/blob/master/mingw-w64-fribidi/0001-Missing-files-in-the-tarball.patch

**Comment by dov** [comment]
Sorry for the delay. Fixed in v1.0.3
